models/src/main/scala/coop/rchain/models/BlockHash.scala

```diff
@@ -0,0 +1,9 @@
+package coop.rchain.models
+
+import com.google.protobuf.ByteString
+
+object BlockHash {
+  type BlockHash = ByteString
+
+  val BlockHashLength = 32
```

Comment:
if this is named Length then you can refer to it as `BlockHash.Length`

Reply:
Good point